components/expenses/ConfirmProcessExpenseModal.tsx

```diff
@@ -14,7 +15,7 @@ import { toast } from '../ui/useToast';
 
 const messages = defineMessages({
   reasonPlaceholder: {
-    defaultMessage: 'e.g. Email Address is wrong',
+    defaultMessage: 'e.g. We never worked with this person.',
```

Comment:
We use the American version for this.

```suggestion
    defaultMessage: 'e.g., We never worked with this person.',
```